src/pages/Player.vue

```diff
@@ -4,7 +4,10 @@
     <video-skeleton v-if="!isVideoIdValid"></video-skeleton>
     <div v-else class="flex relative shadow-lg">
       <!-- fullscreen button overlay -->
-      <div class="z-50 absolute breakpoint-500:hidden w-full h-full bg-transparent">
+      <div
+        v-if="showItemModal"
+        class="z-50 absolute bp-50:hidden w-full h-full bg-transparent"
```

Comment:
bp-500?
have you tested it?

Reply:
Lol arey
haan test toh kia tha fullscreen bug ke lie
Dont know kaise chal gaya

Reply:
Oh, <500px mein I tested where it was working,
> 500px mein it was still showing which I didn't test :P